roles/routeros_common/vars/S2.Triangle.hamwan.net_settings.yml

```diff
@@ -0,0 +1,72 @@
+---
+# vars file for routeros settings
+
+routeros_facts:
+  - name: routeros_version
+    fact: routeros_version
+    query_command: '/system resource print'
+    pattern: '.*version: (\S*).*'
+
+# Settings only for RouterOS 6 and 7
+simple_settings:
+  - name: logging_destination
+    query_command: '/system logging action print detail where name=remote'
+    desired_value: 'remote=44.25.0.8'
+    pattern: '.*remote=(\S*).*'
+    set_command: '/system logging action set [find name=remote] remote=44.25.0.8'
+    missing_ok: false
+
+  - name: snmp_community_addresses
+    query_command: '/snmp community print detail where name=hamwan'
+    desired_value: 'addresses=44.24.240.0/20,44.25.0.0/16'
```

Comment:
This is not the desired value.  Fix.

Reply:
Also extract out the PSDR specific values into a file outside the role definition.